docs/sdk/src/reference_docs/omni_node.rs

```diff
@@ -177,9 +177,35 @@
 //! [This](https://github.com/paritytech/polkadot-sdk/issues/5565) future improvement to OmniNode
 //! aims to make such checks automatic.
 //!
+//! ### Runtime conventions
+//!
+//! Given OmniNode purpose is to be compatible with most Polkadot parachains, we thought it is
+//! also achievable to guide how the developed parachain's runtimes should be composed. The
+//! runtimes should contain a set the basis for a functional parachain in the context of any
+//! node. We check for these pallets at OmniNode start-up, parsing through the runtime metadata.
+//! The checks are best effort, given the runtime code is custom, and can be considered as lint
+//! rules which will generate warning level logs in the Omni Node log file, when not complied with.
```

Comment:
```suggestion
//! The OmniNode needs to make some assumptions about the runtime. During startup, the node fetches the runtime metadata
//! and asserts that the runtime represents a compatible parachain.
//! The checks are best effort and will generate warning level logs in the Omni Node log file on failure.
```

Nit: I think we should be a bit more concise here.

Reply:
Updated as suggested: [0ba6ecb](https://github.com/paritytech/polkadot-sdk/pull/6450/commits/0ba6ecb244c1ae43bfc6262e2c1143bf3f735410)